include/boost/geometry/extensions/gis/io/wkb/detail/ogc.hpp

```diff
@@ -107,99 +107,61 @@ struct ewkt_policy
 {
 };
 
-template
-<
-    typename Geometry, 
-    typename CheckPolicy = ogc_policy, 
-    typename Tag = typename tag<Geometry>::type
->
-struct geometry_type : not_implemented<Tag>
-{
-};
-
-template <typename Geometry, typename CheckPolicy>
-struct geometry_type<Geometry, CheckPolicy, point_tag>
+template <typename Geometry,
+          geometry_type_ogc::enum_t OgcType,
```

Comment:
What do you mean? `enum_t` is a name of a type defined in `geometry_type_ogc`. Do you think that the name of a type should be changed to something else? Or should it be defined in a global scope? Is this is a question to the original author?